acceptance/tests/sameness/sameness_test.go

```diff
@@ -150,6 +150,12 @@ func TestFailover_Connect(t *testing.T) {
 				keyCluster03a: {name: peerName3a, context: env.Context(t, 3), hasServer: true, locality: localityForRegion(cluster03Region)},
 			}
 
+			// Set primary clusters per cluster
```

Comment:
This is where the "meat" of the refactor changes are. The partition cluster (`cluster-01b`) needs to use the primary cluster for DNS operations.